README.md

```diff
@@ -102,3 +102,18 @@ Contributing
 =================
 
 Feel free to fork and send us pull requests. Here are the [guidelines for contribution](https://github.com/Murali-group/GraphSpace/blob/master/CONTRIBUTING.md) in GraphSpace.
+
+
+Contact
+=================
+
+If you have questions or suggestions about GraphSpace, please contact
+
+- **T.M Murali ([@tmmurali](https://github.com/tmmurali))**
```

Comment:
Please change my name to "T. M. Murali" (with a period after the "M").

Reply:
Sorry.I have made the correction.